plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/part/DataMapperDiagramEditor.java

```diff
@@ -600,6 +609,45 @@ private void updateSchema() {
 		reloadDataMapperTestWindow(getInputSchemaType(), getOutputSchemaType());
 	}
 
+    /**
+     * Compares the schema before updating with the new schema to check whether there are any changes
+     * 
+     * @param modifiedSchema - changed schema
+     * @throws ParseException
```

Comment:
Update java doc comments

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/1063